asc.jl

```diff
@@ -19,7 +19,10 @@ Nx, Ny, Nz = 128, 128, 64
 
 decay = Nz/Lz*2
 
-stretched_grid(k) = -(Lz*(tanh(decay*(k-Nz-0.5))))+(Lz*tanh(decay*(-Nz+1-0.5)));
+
+stretched_grid(k) = -(-(Lz*(tanh(decay*(k+Nz+0.5)))-(Lz*tanh(decay*(2*Nz+1.5))))/(-(Lz*(tanh(decay*(Nz+1.5)))-(Lz*tanh(decay*(2*Nz+1.5)))))*Lz)
```

Comment:
```suggestion
stretched_grid(k) = Lz * (tanh(decay * (2Nz + 3/2)) - tanh(decay * (Nz + 1/2 + k))) / 
                            (tanh(decay * (Nz + 3/2)) - tanh(decay * (2Nz + 3/2)))
```

tried to clean it up a bit

Reply:
Also, I suggest after we create the grid we add

```Julia
fig = Figure()
ax = Axis(fig[1, 1], ylabel = "Depth (m)", xlabel = "Vertical spacing (m)")
lines!(ax, grid.Δzᵃᵃᶜ[1:grid.Nz], grid.zᵃᵃᶜ[1:grid.Nz])
scatter!(ax, grid.Δzᵃᵃᶜ[1:Nz], grid.zᵃᵃᶜ[1:Nz])
current_figure()
```

even if we have it commented out! This way one can visualize the stretched grid spacing easily.

Reply:
Done :)